joss/paper.md

```diff
@@ -191,10 +191,31 @@ hydrological forecasting, drought prediction, and climate/remote-sensing
 pipelines; physics plugins include high-energy physics, radio astronomy, lattice
 quantum chromodynamics (QCD), and gravitational-wave/glitch analysis. Packaging
 these as `itwinai` plugins enables reproducible, shareable workflows that run
-consistently on hardware ranging from personal computers to HPC systems. The full list of `itwinai`
-plugins can be found at [this
+consistently on hardware ranging from personal computers to HPC systems. The
+full list of `itwinai` plugins can be found at [this
 link](https://itwinai.readthedocs.io/latest/getting-started/plugins-list.html).
 
+# Reproducibility
+
+The scalability results in this paper were obtained on the JUWELS Booster GPU
+partition at the Jülich Supercomputing Centre [@JUWELS], a SLURM-managed system.
+For the Virgo scalability study (\autoref{fig:speedup},
+\autoref{fig:compvsother}), `itwinai` was driven by YAML configurations that
+specify dataset, model, optimizer, batch size, and number of epochs. The exact
+configuration files, SLURM scripts, and commands used to generate these figures
+are stored in the `itwinai` repository inside `joss/reproducibility`.
```

Comment:
Would recommend have a hyperlink here and make sure it won't change in the future